app/controllers/course/courses_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Course::CoursesController < Course::Controller
   def show #:nodoc:
+    @activities = recent_activities(20)
```

Comment:
...not quite understand. I think we just need to load recent activities in course show page... Will we show activities in other pages?